.env

```diff
@@ -33,9 +33,13 @@ COMPOSE_DOCKER_CLI_BUILD=1
 DOCKER_BUILDKIT=1
 BUILDKIT_INLINE_CACHE=1
 
+# different architecture notations
+ARCH=amd64
+ARCH_ALIAS=x86_64
+ARCH_SHORT_ALIAS=x64
```

Comment:
Can the aliases be deduced automatically from ARCH? It would be more user-friendly.

Reply:
Ha, nevermind, I see you did this below :-)

Reply:
That's done by archery, but placing the defaults in `.env` enables the usage of docker-compose without archery.